style.css

```diff
@@ -30,69 +19,82 @@ body > div {
 h1 {
     font-size: 24px;
     font-weight: 950;
-    color: #F9FAF8;
-    margin-right: auto;
+    color: #f9faf8;
+}
+
+.nav {
+    display: flex;
+    gap: 24px;
 }
 
 .heroMainText {
     font-size: 48px;
     font-weight: bolder;
-    color: #F9FAF8;
+    color: #f9faf8;
 }
 
 .top {
     display: flex;
     padding-top: 24px;
-    justify-content: end;
+    justify-content: space-between;
     gap: 32px;
+    align-items: center;
 }
 
 h2 {
     font-size: 48px;
-    color: #F9FAF8;
+    color: #f9faf8;
     font-weight: 1000;
 }
 
-.topHalfSplit {
+.top-half-split {
     display: flex;
-    gap: 25px;
+    flex-wrap: wrap;
```

Comment:
This allows the split "columns" to wrap and stack at smaller breakpoints i.e. mobile:

Before:

![CleanShot 2024-05-16 at 11 30 06](https://github.com/VamanaPed/LandingPage/assets/199204/1d1d837f-fe60-4d0e-b8ef-80d73a408622)

After:

![CleanShot 2024-05-16 at 11 28 14](https://github.com/VamanaPed/LandingPage/assets/199204/ec1a6501-c024-4c0d-870c-1882d255292c)

You'll learn more about responsive design and layout further on, but something to keep in mind.